apollo-federation/src/operation/mod.rs

```diff
@@ -3032,6 +3032,7 @@ impl SelectionSet {
         self.selections.values().any(|s| s.has_defer())
     }
 
+    /// - `self` must be fragment-spread-free.
```

Comment:
a triple slash is a [doc-level comment](https://doc.rust-lang.org/reference/comments.html#doc-comments) that's meant to show up in docs.rs. since this is a private function we don't need it.

```suggestion
    // - `self` must be fragment-spread-free.
```